tracer/tracer.go

```diff
@@ -255,6 +287,36 @@ func AddEvent(event *protocol.Event) {
 	GlobalTracer.AddEvent(event)
 }
 
+func (tracer *epsagonTracer) verifyLabel(label EpsagonLabel) bool {
+	var valueSize = 0
+	switch label.value.(type) {
+	case int, float64:
+		valueSize = strconv.IntSize
+	case string:
+		valueSize = len(label.value.(string))
+	default:
+		if tracer.Config.Debug {
+			log.Println("EPSAGON DEBUG: type %v is not supported for labels. Supported types are: int, float, string", label.value)
+		}
+		return false
+	}
+	if len(label.key)+valueSize+tracer.labelsSize > MaxLabelSize {
```

Comment:
would rename MaxLabelSize to MaxLabelsSize (Label -> Labels)
add spaces between the +